docs/configuration.md

```diff
@@ -1474,6 +1474,13 @@ Apart from these, the following properties are also available, and may be useful
     the new Kafka direct stream API.
   </td>
 </tr>
+<tr>
+  <td><code>spark.streaming.ui.retainedBatches</code></td>
+  <td>1000</td>
```

Comment:
This is just like `spark.ui.retainedJobs` and `spark.ui.retainedStages` and should be exposed. I think we can merge this after #5533.